A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -774,47 +790,68 @@ class CDUPerformancePage {
                 CDUPerformancePage.ShowAPPRPage(mcdu);
             }
         };
-        mcdu.onRightInput[3] = () => {
-            mcdu.setPerfApprFlaps3(true);
-            mcdu.updatePerfSpeeds();
-            CDUPerformancePage.ShowAPPRPage(mcdu);
-        };
         mcdu.onRightInput[4] = () => {
-            mcdu.setPerfApprFlaps3(false);
+            mcdu.setPerfApprFlaps3(!mcdu.perfApprFlaps3);
             mcdu.updatePerfSpeeds();
             CDUPerformancePage.ShowAPPRPage(mcdu);
         };
-        let finalCell = "-----";
+
+        let finalCell = "";
         const approach = mcdu.flightPlanManager.getApproach();
         if (approach && approach.name) {
-            finalCell = Avionics.Utils.formatRunway(approach.name);
+            finalCell = `{green}${Avionics.Utils.formatRunway(approach.name).replace(/ /g, "")}{end}`;
         }
-        let mdaCell = "[ ]";
+
+        const baroRadio = NXDataStore.get("CONFIG_BARO_RADIO", "0") === "1";
```

Comment:
I'd make a variable for this and set it to the selected state on init and when changed on the options page.